cmd/kube-apiserver/app/options/plugins.go

```diff
@@ -83,4 +89,46 @@ func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
 	setdefault.Register(plugins)
 	resize.Register(plugins)
 	pvcprotection.Register(plugins)
+
+	// Set recommended plugins order
+	recommendedOrder := []string{
+		deny.PluginName,
+		admit.PluginName,
+		autoprovision.PluginName,
+		lifecycle.PluginName,
+		initialization.PluginName,
+		mutatingwebhook.PluginName,
```

Comment:
this order is not the one at https://kubernetes.io/docs/admin/admission-controllers/#is-there-a-recommended-set-of-admission-controllers-to-use @caesarxuchao where should mutating webhook be in the chain?

Reply:
@caesarxuchao ,I am not sure `MutatingAdmissionWebhook` 's relative order with other kinds of mutating plugins.

Reply:
We said we wanted the MutatingAdmissionWebhook to be the last in the mutating plugins.

Reply:
I see.